frontend/src/scenes/onboarding/OnboardingProductIntroduction.tsx

```diff
@@ -0,0 +1,250 @@
+import './onboarding.scss'
+
+import { IconCheck, IconMap, IconMessage, IconStack } from '@posthog/icons'
+import { LemonButton, Link, Spinner } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { WavingHog } from 'lib/components/hedgehogs'
+import React from 'react'
+import { convertLargeNumberToWords } from 'scenes/billing/billing-utils'
+import { billingProductLogic } from 'scenes/billing/billingProductLogic'
+import { ProductPricingModal } from 'scenes/billing/ProductPricingModal'
+import { getProductIcon } from 'scenes/products/Products'
+import { SceneExport } from 'scenes/sceneTypes'
+import { urls } from 'scenes/urls'
+
+import { BillingProductV2Type, BillingV2FeatureType, ProductKey } from '~/types'
+
+import { onboardingLogic } from './onboardingLogic'
+
+export const scene: SceneExport = {
+    component: OnboardingProductIntroduction,
+    logic: onboardingLogic,
+}
+
+export const Feature = ({ name, description, images }: BillingV2FeatureType): JSX.Element => {
+    return images ? (
+        <li className="text-center">
+            <div className="mb-2 w-full rounded">
+                <img src={images.light} className="w-full rounded" />
```

Comment:
should include logic for dark image.

Reply:
We don't have dark mode images yet, I figured we can actually add them later.